src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -846,6 +842,8 @@
 					CHIP_HAVE_CONFIG_H,
 					"CHIP_ADDRESS_RESOLVE_IMPL_INCLUDE_HEADER=<lib/address_resolve/AddressResolve_DefaultImpl.h>",
 					"$(inherited)",
+					"MTR_NEWLY_AVAILABLE=",
```

Comment:
How come these need to be defined in the project file?

Reply:
I'm using these in various framework eaders.  So I need to do one of:

1. Define them in every header I use them in.
2. Define them in some header that all the framework headers include.
3. Define them in the project file.
4. Something else I did not think of.

For now 3 was the most expedient, but I am very open to other suggestions, as long as they are not item 1.

Reply:
Going to keep this as-is for now, but happy to change to something else in a followup.